optimade/models/jsonapi.py

```diff
@@ -237,7 +240,7 @@ def check_illegal_attributes_fields(cls, values):
 
 
 class Resource(BaseResource):
-    """Resource objects appear in a JSON:API document to represent resources."""
+    """Resource objects appear in a JSON API document to represent resources."""
```

Comment:
The `:` is taken directly from the JSON API spec, see [here](https://jsonapi.org/format/1.0/#document-resource-objects).

Reply:
Fair, we don't use that anywhere in the specification though, so think I'd rather keep it colon-less.